plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -234,12 +237,15 @@ export class ExpressInstrumentation extends InstrumentationBase<ExpressInstrumen
         }
 
         let spanHasEnded = false;
+        // TODO: Fix router spans (getRouterPath does not work properly) to
```

Comment:
Can you expand on what you mean by not working properly or create an issue so this doesn't get lost?

Reply:
You can see in the [nested routers test](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/instrumentation-express-v0.47.0/plugins/node/opentelemetry-instrumentation-express/test/express.test.ts#L680-L722) that the created router spans never include [the `/posts` section of the route](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/instrumentation-express-v0.47.0/plugins/node/opentelemetry-instrumentation-express/test/fixtures/use-express-nested-router.mjs#L50-L64). If I were to propagate context and nest them we'd end up with `/api/user/:id` -> `/:postId` -> `/api/user/posts/:postId` which is really confusing.